news/+misc.misc.rst

```diff
@@ -0,0 +1 @@
+Misc fixes, improvements and micro optimizations.
```

Comment:
Don't you think it would be better to list the fixes and optimizations made?

Reply:
Well, this news fragment is for the things that the end-users probably won't care about. If you think that any of these changes deserve their own news  fragment, we may add it.

Reply:
> Well, this news fragment is for the things that the end-users probably won't care about. If you think that any of these changes deserve their own news fragment, we may add it.

Good point lol